runbotics/runbotics-orchestrator-ui/src/components/HistoryTable/HistoryTable.columns.tsx

```diff
@@ -10,7 +10,11 @@ import { getProcessInstanceStatusColor } from '../../utils/getProcessInstanceSta
 import { convertToPascalCase } from 'src/utils/text';
 import useTranslations from 'src/hooks/useTranslations';
 
-const useProcessInstanceColumns = (): Column<IProcessInstance>[] => {
+type ProcessInstanceColumn<D extends object = {}> = Column<D> & {
```

Comment:
this should be type created in components/table and any other Column imports should be change to this type (single source of truth)

Reply:
done